src/components/views/rooms/BasicMessageComposer.js

```diff
@@ -212,6 +211,7 @@ export default class BasicMessageEditor extends React.Component {
             if (type === "cut") {
                 // Remove the text, updating the model as appropriate
                 replaceRangeAndMoveCaret(range, []);
+                this._modifiedFlag = true;
```

Comment:
I think this should be set before mutating the editor model with `replaceRangeAndMoveCaret`.

Reply:
ah yep, good point.